kola/tests/misc/grub.go

```diff
@@ -140,8 +140,22 @@ var (
 
 coreos:
   units:
-    - name: update-engine.service
-      mask: true
+    - name: stop-and-mask-update-engine.service
+      enabled: true
+      contents: |
+        [Unit]
+        Description=Stop Update Engine
+        Conflicts=update-engine.service
+        Before=update-engine.service
+
+        [Service]
```

Comment:
```suggestion
        [Service]
        Type=oneshot
        RemainAfterExit=true
```
Not really required but in case this gets copied elsewhere I think it's better to have this set to avoid that the service gets started multiple times